backend/FwLite/MiniLcm/Models/WritingSystemId.cs

```diff
@@ -30,6 +30,7 @@ public override void WriteAsPropertyName(Utf8JsonWriter writer, WritingSystemId
 public readonly record struct WritingSystemId: ISpanFormattable, ISpanParsable<WritingSystemId>
 {
     public string Code { get; init; }
+    public bool IsAudio => Code.EndsWith("-audio", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
So, there seems to be an edge case. I'm not sure how critical it is.
But you can add a variation of a voice WS and then it doesn't end with `-audio`

This is an audio WS that ends with `-audio-var`:
![image](https://github.com/user-attachments/assets/9006e1f6-c38d-431b-80d2-a0aeaefe33cc)

Reply:
Though, just also checking for contains `-audio-` is probably a win.
(or syncing `CoreWritingSystemDefinition.IsVoice` into MiniLcm 🤷)

Reply:
yeah this is what LF does so I'll just copy that for now

Reply:
Actually, as far as I can tell, LF does `includes('-audio')` not `EndsWith`
![image](https://github.com/user-attachments/assets/d8795bfc-a2c3-4c5b-8929-ac11278a1ec5)

Reply:
we're now doing the same thing as `CoreWritingSystemDefinition.IsVoice` sadly I had to copy the code in IsAudio, but I was able to reuse the parsing done, so it's pretty simple.